apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/AppNamespaceService.java

```diff
@@ -16,17 +16,25 @@
  */
 package com.ctrip.framework.apollo.portal.service;
 
+import com.ctrip.framework.apollo.common.dto.ClusterDTO;
```

Comment:
It looks like there is no logic change in this file, so I suppose these imports are not used?